app/presenters/mainstream_browse_page_presenter.rb

```diff
@@ -9,6 +9,20 @@ def format
   def links
     super.merge(
       "related_topics" => @tag.topics.order(:title).map(&:content_id),
+      "top_level_browse_page" => top_level_browse_page_id,
+      "top_level_browse_pages" => @tag.class.sorted_parents.map(&:content_id),
+      "second_level_browse_pages" => @tag.class.second_level_pages.map(&:content_id),
```

Comment:
I don't think this is correct.  It'll return all second-level pages that exist, it needs to only return the ones that are children of the current top-level page.